src/content/docs/browser/browser-integrations/wordpress-fullstack-integration.mdx

```diff
@@ -0,0 +1,160 @@
+---
+
+
+import infrastructureWordPressIntegrationDashboard from 'images/infrastructure_screenshot-full_wordpress-dashboard.webp'
+
+Our WordPress integration monitors the performance of your WordPress web application, helping you diagnose issues in your application and optimize your code. Our WordPress integration makes use of our PHP, Apache, and MySQLmonitoring agents, and gives you a pre-built dashboard with your most important WordPress metrics like transactions, visitors and call duration.
+
+<img
+  title="WordPress dashboard"
+  alt="A screenshot depicting the wordpress dashboard"
+  src={infrastructureWordPressIntegrationDashboard}
+/>
+
+<figcaption>
+After setting up our WordPress integration, we give you a dashboard for your WordPress metrics. 
+</figcaption>
+
+<Steps>
+<Step>
+### Install the relevant agents[#infra-install]
+
+1.  Follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent) for the host containing your WordPress app.
+2. Open [PHP quickstart installation](https://newrelic.com/instant-observability/php).
+Click on install button to start the PHP agent installation.
+3. Open [MySQL quickstart installation](https://newrelic.com/instant-observability/mysql).
+Click on install button to start the MySQL agent installation.
+4. Open [Apache quickstart installation](https://newrelic.com/instant-observability/apache).
+Click on install button to start the Apache agent installation.
+</Step>
+
+<Step>
+### NRI-Flex Configuration
+
+Installing the infrastructure agent will automatically install flex binary. 
+
+To create a flex configuration file follow these steps:
+
+1.  Create a file named `read-wordpress-files-config.yml` in this path:
+
+  ```shell
+  /etc/newrelic-infra/integrations.d
+  ```
+
+2.  Update the `read-wordpress-files-config.yml` with the below details.
+
+    *   INSERT_EVENT_TYPE: You can consider `event_type` as a New Relic database table that you can query using NRQL (for example, WPDirectories).
+    *   INSERT_WORDPRESS_PATH: It takes the directory of the WordPress application (for example, /srv/www/wordpress/*).
+
+3.  Use the below configuration file:
+
+  ```
+  integrations:
+    - name: nri-flex
+      interval: 180s
+      config:
+        name: linuxDirectorySize
+        apis:
+          - event_type: INSERT_EVENT_TYPE
+            commands:
+              - run: du INSERT_WORDPRESS_PATH
+                split: horizontal
+                set_header: [dirSizeKB, dirName]
+                regex_match: true
+                split_by: (\d+)\s+
+  ```  
+</Step>
+
+<Step>
+
+### Configure WordPress to expose debug logs
+
+1. Open your WordPress application and then open `wp-config.php` file
+2. Update the file with the below values as it does log all errors, notices, and warnings to a file called debug.log in the wp-content directory.
```

Comment:
review for readability, quite a few grammatical errors (e.g., "the below values")